recipes-bsp/u-boot/u-boot_2020.04.bb

```diff
@@ -0,0 +1,30 @@
+require recipes-bsp/u-boot/u-boot-common.inc
+require recipes-bsp/u-boot/u-boot.inc
+
+DEPENDS += " bc-native dtc-native swig-native python3-native flex-native bison-native "
+DEPENDS_append_sun50i = " atf-sunxi "
+
+SRCREV = "36fec02b1f90b92cf51ec531564f9284eae27ab4"
```

Comment:
If we not set BBMASK I think that only `require` and SRCREV should be necessery in this recipe